pages/index.css

```diff
@@ -2,3 +2,37 @@
 @import url(../vendor/fonts/fonts.css);
 @import url(../blocks/page/page.css);
 @import url(../blocks/page/__container/page__container.css);
+
+
+@import url(../blocks/header/__menu-burger/header__menu-burger.css);
+@import url(../blocks/header/header.css);
+@import url(../blocks/header/__nav-list/header__nav-list.css);
+@import url(../blocks/header/__navigation-link/header__navigation-link.css);
+@import url(../blocks/header/__navigation-link/_active/header__navigation-link_active.css);
+@import url(../blocks/header/__container/header__container.css);
+@import url(../blocks/header/__menu-icon/header__menu-icon.css);
+
+
+
+@import url(../blocks/intro/intro.css);
+
+
+@import url(../blocks/logo/logo.css);
+@import url(../blocks/logo/__icon/_screen/logo__icon_screen_mobile.css);
```

Comment:
@NazarPauk You have neatly implemented all the code. Good job!

I thought its good to point out here for you to remember that **it is better to import BEM modifiers after the block or element they are modifying the style for**. Here, in your case it works because you're not overwriting any styles in the modifiers. So, I'll not ask you to change anything but remember, there will be cases where you will be overwriting the CSS styles of a block or an element in its modifier, in that case, the import order is very very important. And the above will not work as intended.

Reply:
Thanks for pointing out the possibilities for code improvement!❤️